### PR TITLE
fix(cts): add e2e for ingestion API

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

Add a bit more tests for ingestion API, it can be flaky if someones create a authentication however

### CODE REVIEW

**Review by shortcuts — APPROVED** [review]
love it!

**Comment by millotp** [comment]
I have very low confidence for this PR

File: tests/CTS/requests/ingestion/getAuthentications.json [hunk 1]
```diff
@@ -27,12 +27,42 @@
...
+            "createdAt": "2024-01-08T16:47:31Z",
+            "updatedAt": "2024-01-08T16:47:31Z"
+          },
+          {},
+          {},
+          {},
+          {},
+          {},
+          {},
+          {}
```
Comment (Fluf22) [inline tests/CTS/requests/ingestion/getAuthentications.json:63]: Is this to only validate the first element thanks to the `union` helper?

**Review by Fluf22 — APPROVED** [review]

File: tests/CTS/requests/ingestion/getAuthentications.json (hunk 1 above)
Comment (millotp) [reply tests/CTS/requests/ingestion/getAuthentications.json:63]: exactly, maybe we should make the helper allow for partial match in lists ? like skip remaining elements by looping only over `min(received.length, expected.length)` ?

Comment (Fluf22) [reply tests/CTS/requests/ingestion/getAuthentications.json:59]: That's a good idea

**Review by Fluf22 — APPROVED** [review]
Amazing job 🙌🏻

**Pull request merged**